src/benchmarks/real-world/PowerShell.Benchmarks/Categories.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace MicroBenchmarks
+{
+    public static class Categories
+    {
+        /// <summary>
+        /// Benchmarks belonging to this category are executed for CI jobs.
+        /// </summary>
+        public const string Components = "Components";
+
+        /// <summary>
+        /// Benchmarks belonging to this category are executed for CI jobs.
+        /// </summary>
+        public const string Engine = "Engine";
+
+        /// <summary>
+        /// Benchmarks belonging to this category are targeting internal APIs.
+        /// </summary>
+        public const string Internal = "Internal";
```

Comment:
Are the Internal and Public categories specifically for denoting the API availability or should they also be extended to the run environment? i.e. only run public publicly and internal internally?

Reply:
@adityapatwardhan - Perhaps you could shed some light on this one. We are copying PowerShell's benchmarks over to dotnet/performance verbatim

Reply:
We should add at least the public run verification to this, I will make a PR against the branch to add the runs. 👍

Reply:
@TIHan made the PR here: https://github.com/TIHan/performance/pull/1

Reply:
@LoopedBard3 Merged the change.

Reply:
Looks like I made a typo in the PR in the azure yaml. "src/benchmarks/real-world/Powershell.Benchmarks/PowerShell.Benchmarks.csproj" needs to have the first Powershell changed to PowerShell. I can make a PR for this, but it may be a quicker change for you to push. There should be 2 locations in the azure-pipelines.yml file.

Reply:
Ok, I fixed it.